object_detection/eval_util.py

```diff
@@ -356,7 +358,11 @@ def run_checkpoint_once(tensor_dict,
   """
   if save_graph and not save_graph_dir:
     raise ValueError('`save_graph_dir` must be defined.')
-  sess = tf.Session(master, graph=tf.get_default_graph())
+  session_config = tf.ConfigProto()
```

Comment:
Default session_config to None, and create if per_process_gpu_memory_fraction is passed in.